.github/labeler.yml

```
@@ -5,6 +5,14 @@
 "[JSON]":
   - "**/*.json"
 
+"Game: Achievements / Conducts / Scores":
+  - data/json/scores.json
+  - data/json/statistics.json
+  - data/json/achievements.json
+  - data/json/conducts.json
+  - "**/achievement.*"
```

```suggestion
  - "**/scores**"
  - "**/statistics**"
  - "**/achievements**"
  - "**/conducts**"
```
To cover everything
> fd --type f | rg ".+/scores"
./data/json/scores.json
./src/scores_ui.h
./src/scores_ui.cpp

> fd --type f | rg ".+/statistics"
./data/mods/Magiclysm/achievements/statistics.json
./data/mods/DinoMod/achievements/statistics.json
./data/mods/Aftershock/achievements/statistics.json
./data/json/statistics.json

> fd --type f | rg ".+/scores"
./src/scores_ui.h
./src/scores_ui.cpp
./data/json/scores.json

> fd --type f | rg ".+/achievement"
./src/achievement.h
./src/achievement.cpp
./data/mods/Magiclysm/achievements/spells.json
./data/mods/Magiclysm/achievements/statistics.json
./data/mods/DinoMod/achievements/statistics.json
./data/mods/DinoMod/achievements/achievements.json
./data/mods/Aftershock/achievements/statistics.json
./data/mods/Aftershock/achievements/achievements.json
./data/json/achievements.json
./lang/string_extractor/parsers/achievement.py

> fd --type f | rg ".+/conducts"
./data/json/conducts.json